services/evaluator/scripts/run_tests.py

```
@@ -0,0 +1,121 @@
+        f"Specific Test: {test_path}"
+    )
+
+
+def lint_code():
+    """Run code linting (if available)"""
+    try:
+        return run_command(
+            "python -m flake8 app/ tests/",
+            "Code Linting"
+        )
+    except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)